fuel-vm/src/interpreter/executors/main.rs

```diff
@@ -1042,16 +1044,11 @@ where
             &self.balances,
             gas_price,
         )?;
+        self.update_transaction_outputs()?;
+        *self.tx.as_script_mut().unwrap().receipts_root_mut() = self.receipts.root();
```

Comment:
Whiel it is true that we run only script, I still prefer to sue `if let Some(script)`=)

Reply:
At the top of the function we error if `as_script()` cannot be used. We only access it again like this to satisfy borrow checker.

Reply:
I agree. I don't think we should `unwrap` anywhere in our prod code. I prefer the type-safe approach too--`expect` is more acceptable than `unwrap` but it's fragile still.

Reply:
Fixed in 690a381e1.